elfpy/eth/rpc_interface.py

```diff
@@ -34,3 +34,28 @@ def get_account_balance(web3: Web3, account_address: str) -> int | None:
     if hex_result is not None:
         return int(hex_result, base=16)  # cast hex to int
     return None
+
+
+def get_wait_for_new_block(web3: Web3) -> bool:
```

Comment:
sorry i misled you.  this is not really a generalized function that belongs in rpc_interface, its pretty opinionated.  should just be a private helper function in the file its being used.